src/loaders/TiffLoader.ts

```diff
@@ -49,6 +48,24 @@ class OMEDims {
   channelnames: string[] = [];
 }
 
+function getDtype(omepixeltype: string): NumberType {
+  const mapping: Record<string, NumberType> = {
+    uint8: "uint8",
+    uint16: "uint16",
+    uint32: "uint32",
+    int8: "int8",
+    int16: "int16",
+    int32: "int32",
+    float: "float32",
+  };
+  const dtype = mapping[omepixeltype];
+  if (dtype === undefined) {
+    // TODO consider throwing an error?
+    return "uint8";
```

Comment:
Might be good to log this at least? "unrecognized TIFF pixel type; defaulting to 'uint8'" so that if that guess is wrong and causes an error later, folks are more likely to know where to actually look